### PR TITLE
content(Upload): updated based on figma content updates after usability testing

### LINKED ISSUE #836: [Upload] Content Update
Based on usability testing, language content in the figma was updated. This will need to be updated in the app

### PR DESCRIPTION
**Pull request by shindigira**
closes #836

## Changes
- content(Upload): updated alert and buttons (e.g. 'Continue to next step') based on figma content updates after usability testing

## How To Test
- Will deploy to dev pub -- compare the figma with what is deployed
- Upload - warnings, errors, success csv's to see the updated messages

[logic-errors_small.csv](https://github.com/user-attachments/files/16501790/logic-errors_small.csv)
[logic-warnings_small.csv](https://github.com/user-attachments/files/16501791/logic-warnings_small.csv)
[pass_123456789TESTBANK456_small.csv](https://github.com/user-attachments/files/16501792/pass_123456789TESTBANK456_small.csv)

### CODE REVIEW

File: src/pages/Filing/FilingApp/FileSubmission.data.tsx [hunk 1]
```diff
@@ -190,11 +189,11 @@ export const fileSubmissionValidationStatus: Record<
   string
 > = {
   [FileSubmissionState.VALIDATION_SUCCESSFUL]:
-    'The validation checks returned no errors or warnings',
+    'Navigate through the validation results pages to continue to the required next step.',
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileSubmission.data.tsx:192]: Change to: 
"Navigate through the validation results pages to continue."

File: src/pages/Filing/FilingApp/FileSubmission.data.tsx [hunk 2]
```diff
@@ -190,11 +189,11 @@ export const fileSubmissionValidationStatus: Record<
...
   [FileSubmissionState.SUBMISSION_ACCEPTED]:
-    'The validation checks returned no errors or warnings',
+    'Navigate through the validation results pages to continue to the required next step.',
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileSubmission.data.tsx:194]: Change to: 
"Navigate through the validation results pages to continue."

File: src/pages/Filing/FilingApp/FileSubmission.data.tsx [hunk 3]
```diff
@@ -190,11 +189,11 @@ export const fileSubmissionValidationStatus: Record<
...
   [FileSubmissionState.VALIDATION_WITH_WARNINGS]:
-    'The validation checks returned warnings',
+    'Your file contained no errors but did contain warnings. Navigate through the error validation results pages to continue to the required next step.',
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileSubmission.data.tsx:196]: Change to: 
"Your file contains no errors but does contain warnings. Navigate through the error validation results pages to continue."

File: src/pages/Filing/FilingApp/FileSubmission.tsx [hunk 4]
```diff
@@ -373,7 +373,7 @@ export function FileSubmission(): JSX.Element {
                   <>
                     <FieldGroupDivider />
                     {/* Upload Status Section */}
-                    <Heading type='3'>Status</Heading>
+                    <Heading type='3'>Upload Summary</Heading>
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileSubmission.tsx:376]: Change to: 
"Upload summary"

File: src/pages/Filing/FilingApp/FileDetailsValidation.tsx [hunk 5]
```diff
@@ -26,7 +26,7 @@ function FileDetailsValidation({
   const validationStatusMessage: string =
     hasWarnings &&
     dataGetSubmissionLatest.state === FileSubmissionState.SUBMISSION_ACCEPTED
-      ? 'Your file contains no errors and warnings have been verified'
+      ? 'Your file contained no errors and warnings have been verified. Navigate through the validation results pages to continue to the required next step.'
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileDetailsValidation.tsx:29]: Change to: 
"Your file contains no errors and warnings have been verified. Navigate through the validation results pages to continue."

**Review by natalia-fitzgerald — CHANGES_REQUESTED** [review]
@shindigira 
Thanks for sharing these changes. I have a few content revisions that I have noted here.

File: src/pages/Filing/FilingApp/FileSubmission.data.tsx [hunk 6]
```diff
@@ -125,12 +125,8 @@ export function MustUploadFirstAlert(): JSX.Element {
...
       aria-live='polite'
       aria-atomic='true'
-      message={
-        <FormParagraph>
-          File upload and validation checks must be completed to save and
-          continue.
-        </FormParagraph>
-      }
+      headingLevel=''
+      message='Your file must be successfully uploaded and validation checks must be performed to continue to the next step.'
```
Comment (natalia-fitzgerald) [inline src/pages/Filing/FilingApp/FileSubmission.data.tsx:129]: @shindigira - There was a change here that I'm not sure was caught. We removed the second "must be" from this message. 

'Your file must be successfully uploaded and validation checks performed to continue to the next step.'

**Review by natalia-fitzgerald — CHANGES_REQUESTED** [review]
@shindigira 
I noted one item that should be updated if it hasn't yet.

**Review by billhimmelsbach — APPROVED** [review]
Looks good!

**Pull request merged**